sys/arm64/arm64/elf_machdep.c

```diff
@@ -386,7 +386,7 @@ reloc_instr_imm(Elf32_Addr *where, Elf_Addr val, u_int msb, u_int lsb)
 
 #if __has_feature(capabilities)
 static void __nosanitizecoverage
-decode_fragment(Elf_Addr *fragment, Elf_Addr relocbase, Elf_Addr *addrp,
+decode_fragment(Elf_Addr *fragment, Elf_Addr relocbase, uintptr_t *addrp,
```

Comment:
This confuses me, relocbase is still an Elf_Addr so this just creates a NULL-derived uintptr_t? That seems like a waste for purecap kernels?

Reply:
Un-ifdef val and use that, or some other temporary? I don't see a reason why this needs to be like this.

Reply:
Fixed